src/extension.ts

```diff
@@ -28,8 +28,7 @@ export function activate(context: ExtensionContext) {
 
 	let serverOptions: Executable = {
 		command: 'java',
-		args: [ '-jar', camelLanguageServerPath],
-		options: {stdio:'pipe'}
```

Comment:
why has that been removed?

Reply:
I don't know exactly why they removed this option. it seems that it is when they introduced some facilities for Node based Language servers.
I found out that it was between 6.0.0.next6 and 6.0.0.next7 but there is no corresponding issue number on the commits.

Reply:
I was just curious.

Reply:
I understand. I was curious too when discovering that it has been removed but wasn't able to find the official reason :-(